_targets/build.project.armv7m7-imxrt106x

```diff
@@ -28,98 +30,63 @@ export PORTS_JANSSON=n
-                "wait 2000")
-
-# Obligatory applications to run Phoenix-RTOS
-USER_SCRIPT=("kernel ${BOOT_DEVICE}"
-             "app ${BOOT_DEVICE} -x dummyfs xip1 ocram2"
-             "app ${BOOT_DEVICE} -x imxrt-multi ocram2 ocram2"
-             "app ${BOOT_DEVICE} -x psh xip1 ocram2"
-             "app ${BOOT_DEVICE} imxrt-flash ocram2 ocram2")
-
-
-b_mkscript_user() {
-	mkdir -p "$PREFIX_BUILD/plo/"
-	rm -f "$PREFIX_BUILD/plo/$NAME_USER_SCRIPT"
-
-	ksz=$((($(wc -c < "${PREFIX_PROG_STRIPPED}phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf") + SIZE_PAGE - 1) & PAGE_MASK))
-	poffs=$((KERNEL_OFFS + ksz))
-
-	{
-		printf "%08x\n" "$MAGIC_USER_SCRIPT"
-		for cmd in "${USER_SCRIPT[@]}"; do
-			args=($cmd)
-			case "${args[0]}"
-			in
-				app)
-					if [ "${args[2]}" = "-x" ]; then
-						name=${args[3]%%;*}
-					else
-						name=${args[2]%%;*}
-					fi
-
-					PROGS+=("$name")
-					psz=$((($(wc -c < "${PREFIX_PROG_STRIPPED}$name") + SIZE_PAGE - 1) & PAGE_MASK))
-					printf "alias %s 0x%x 0x%x\n" "$name" "$poffs" "$psz"
-					((poffs+=psz));;
-
-				kernel)
-					printf "alias %s 0x%x 0x%x\n" "phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf" "$KERNEL_OFFS" "$ksz";;
-			esac;
-			printf "%s\n" "$cmd"
-		done
-
-		printf "\0"
-	} >> "$PREFIX_BUILD/plo/$NAME_USER_SCRIPT"
-
-}
+export BOOT_DEVICE="flash1"
+export NAME_USER_SCRIPT="user.plo"
+export OFFS_USER_SCRIPT=$((0x10a00))
+export MAGIC_USER_SCRIPT=$((0xdabaabad))
 
 
-b_mkscript_preinit() {
-	file="$PREFIX_BUILD/plo/script.plo"
-	mkdir -p "$PREFIX_BUILD/plo/"
-	rm -f "$file"
-
-	{
-		for cmd in "${PREINIT_SCRIPT[@]}"; do
-			printf "%s\n" "$cmd"
-		done
-
-		sz=$(wc -c < "$PREFIX_BUILD/plo/$NAME_USER_SCRIPT")
-		printf "alias %s 0x%x 0x%x\n" "$NAME_USER_SCRIPT" "$OFFS_USER_SCRIPT" "$sz"
-		printf "call %s %s %08x\n" "$BOOT_DEVICE" "$NAME_USER_SCRIPT" "$MAGIC_USER_SCRIPT"
-		printf "\0"
-	} >> "$file"
-}
+# Pre-init script is launched before user script
+export PREINIT_SCRIPT=(
+	"map itcm 0x0 0x40000 rwx"
+	"map dtcm 0x20000000 0x20028000 rw"
+	"map ocram2 0x20200000 0x2027fe00 rwx"
+	"map xip1 0x70000000 0x70400000 rx"
+	"map xip2 0x60000000 0x64000000 rx"
+	"phfs usb0 1.2 phoenixd"
+	"phfs flash0 2.0 raw"
+	"phfs flash1 2.1 raw"
+	"console 0.0")
+
+
+# Production user script contains applications to run Phoenix-RTOS
+export USER_SCRIPT=(
+	"wait 2000"
+	"kernel ${BOOT_DEVICE}"
+	"app ${BOOT_DEVICE} -x dummyfs xip1 ocram2"
+	"app ${BOOT_DEVICE} -x imxrt-multi ocram2 ocram2"
+	"app ${BOOT_DEVICE} -x psh xip1 ocram2"
+	"app ${BOOT_DEVICE} imxrt-flash ocram2 ocram2")
+
+
+# Example of user dev script which call remotely script
+export DEV_USER_SCRIPT=(
+	"echo on"
+	"wait 3000"
+	"call usb0 script-${TARGET}-dev.plo 5a5aa5a5")
+
+
+export REMOTE_USER_SCRIPT=(
+	"5a5aa5a5"
+	"kernel usb0"
+	"app usb0 -x dummyfs ocram2 ocram2"
+	"app usb0 -x imxrt-multi itcm dtcm"
+	"app usb0 -x psh ocram2 ocram2"
+	"copy usb0 imxrt-flash flash1 0x15000 0"
+	"app flash1 imxrt-flash xip1 dtcm"
+	"go!")
 
```

Comment:
Do these arrays need to have an `export` attribute. I guess they are only used in this file.